spyder/plugins/completion/tests/conftest.py

```diff
@@ -13,6 +13,10 @@
 # Third party imports
 from qtpy.QtWidgets import QMainWindow
 
+# PyTest imports
+import pytest
+from pytestqt.qtbot import QtBot
+
```

Comment:
Please revert this change. Without it we can't run our tests individually due to the issue mentioned in the comment below with QtAwesome.

Reply:
Are you sure about this? Neither pytest nor pytestqt.qtbot are initializing anything which you haven't imported above the comment already.

I removed the last commit from HEAD.